browser/BUILD.gn

```diff
@@ -29,6 +29,28 @@ source_set("version_info") {
   ]
 }
 
+source_set("browser_process") {
+  visibility = [
+    "//brave/browser/*",
+    "//chrome/browser/*",
+  ]
+
+  sources = [
+    "brave_browser_process.h"
+  ]
+
+  deps = [
+    "//brave/components/brave_ads/browser/buildflags",
+    "//brave/components/brave_referrals/buildflags",
+    "//brave/components/greaselion/browser/buildflags",
+    "//brave/components/ipfs/buildflags",
+    "//brave/components/speedreader",
+    "//brave/components/tor/buildflags",
+    "//chrome/browser:browser_process",
```

Comment:
Would it make sense to move `"//chrome/browser:browser_process"` into `public_deps` ?

To be clear, I'm not sure myself about that since I like how **not** having it in `public_deps` forces other targets including files from `//chrome/browser:browser_process` to explicitly depend on it, but I'd rather double-check this is the actual purpose, just in case.

Reply:
yea, I'd rather not make it public here